src/sentry/models/options/project_template_option.py

```diff
@@ -1,18 +1,21 @@
 from __future__ import annotations
 
 from collections.abc import Callable, Mapping, Sequence
-from typing import Any, ClassVar
+from typing import TYPE_CHECKING, Any, ClassVar
 
 from django.db import models
 
 from sentry.backup.scopes import RelocationScope
 from sentry.db.models import FlexibleForeignKey, Model, region_silo_model, sane_repr
 from sentry.db.models.fields import PickledObjectField
 from sentry.db.models.manager.option import OptionManager
-from sentry.models.projecttemplate import ProjectTemplate
 from sentry.utils.cache import cache
 
 Value = Any | None
+TProjectOptions = Mapping[str, Value]
```

Comment:
What is the `T` here for? Template?

Reply:
Type | Template. i didn't want to choose and TT looked weird. 🤣